CONTRIBUTING.md

```diff
@@ -31,23 +33,30 @@ We recommend you to use our prebuilt Docker image to start your development work
 
 * From VSCode's command menu, run `Reopen in Container` to open your workspace in one of our pre-built Docker containers. Select the correct container config based on your local accelerator (default to `tpu-contributor` if you are not sure).
```

Comment:
Somehow I couldn't find "Reopen in Container" so I can't verify the whole process.

Reply:
I think @will-cromar  added this section. I've done many things with vscode so I likely had the right extension. Maybe you need this one: [2]?

[2]: https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers

Reply:
I have the extension `Dev Containers`.  Is `Reopen in Container` the correct name?

Reply:
Yes! (and select the appropriate json config)

Reply:
I know what's missing. One should open the pytorch/xla folder first in vscode, then `Reopen in Container` will show up after pressing F1, then choose the config. Could you add the missing info to the doc?

Reply:
Done